modules/setting/cors.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package setting
+
+import (
+	"code.gitea.io/gitea/modules/log"
+	"github.com/go-macaron/cors"
+)
+
+var (
+	// CORSConfig defines CORS settings
+	CORSConfig cors.Options
+	// EnableCORS defines whether CORS settings is enabled or not
+	EnableCORS bool
+)
+
+func newCORSService() {
+	sec := Cfg.Section("cors")
+	// Check cors setting.
+	if !sec.Key("ENABLED").MustBool() {
+		return
+	}
+
+	CORSConfig = cors.Options{
+		Scheme:           sec.Key("SCHEME").String(),
```

Comment:
Could you add these options to the sample configuration in `custom/conf/app.ini.sample`, and also in `docs/content/doc/advanced/config-cheat-sheet.en-us.md`